### DISCUSSION TITLE
Helm charts for hydra v1.11.10

### DISCUSSION THREAD

**Discussion by uap-dev** [ory/.github#26] [Q&A]
I am hoping to take advantage of the custom ttl per client feature available on v1.11.10 https://github.com/ory/hydra/pull/3206. I see that the helm charts have not been updated to this app version yet. Is there a suggestion on how to deploy via helm for v1.11.10?

**Comment by vinckr** [comment]
Hello @TroutZen 
see this issue: https://github.com/ory/k8s/issues/464. 
Also see [this issue (update to 2.0)](https://github.com/ory/k8s/issues/448), the release will come soon.
